packages/editor-ui/src/store.ts

```diff
@@ -738,6 +738,11 @@ export const store = new Vuex.Store({
 			return state.activeWorkflows;
 		},
 
+		activeWorkflowTriggerNodes: (state, getters) => {
```

Comment:
maybe rename this to `workflowTriggerNodes`.. I suggested `active` before because I thought we would also check if nodes are disabled. 
Otherwise update code to only return active (not disabled) nodes

Reply:
Renamaed